packages/blockchain-wallet-v4-frontend/src/modals/Lockbox/LockboxSetup/DeviceSelectStep/index.js

```diff
@@ -13,8 +13,18 @@ class DeviceSelectStepContainer extends React.PureComponent {
     this.props.lockboxActions.changeDeviceSetupStep('setup-type')
   }
 
+  handleRestoreClick = () => {
+    const restoreWindow = window.open(this.props.restoreDeviceLink, '_blank')
```

Comment:
Potentially dangerous, is there a way to pass noopener, noreferrer here?

Reply:
fixed